assignmentTest/splab/Gopistol/invitation/src/main/java/com/splab/invitation/domain/member/entity/Member.java

```diff
@@ -0,0 +1,44 @@
+package com.splab.invitation.domain.member.entity;
+
+import com.splab.invitation.domain.common.BaseEntity;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "member")
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Member extends BaseEntity {
+
+  @Column(nullable = false)
+  private String name;
+
+  @Column(nullable = false, unique = true)
+  private String phone;
+
+  @Column(nullable = false, unique = true)
+  private String email;
+
+
+  @Enumerated(EnumType.STRING)
+  private InvitationStatus status;
+
+  @Builder
+  public Member(String name, String phone, String email, InvitationStatus status) {
+    this.name = name;
+    this.phone = phone;
+    this.email = email;
+    this.status = status;
+  }
+
+  public void updateStatus() {
+    this.status = InvitationStatus.ACTIVATED;
+  }
```

Comment:
조금 더 직관적이고 명시적인 이름이 좋을 것 같아요
```suggestion
  public void activate() {
    this.status = InvitationStatus.ACTIVATED;
  }
```

Reply:
직관적인 이름 고려하겠습니닷